### PR TITLE
Make InteractiveWindow use HACK_ThemeColorFixer

### PR DESCRIPTION
**Pull request by genlu**
Fix issue https://github.com/dotnet/roslyn/issues/4930

@amcasey @dpoeschl @tmat @cston @KevinH-MS @ManishJayaswal

### CODE REVIEW

File: src/VisualStudio/InteractiveServices/Interactive/VsInteractiveWindowPackage.cs [hunk 1]
```diff
@@ -52,6 +52,8 @@ protected override void Initialize()
 
             _componentModel = (IComponentModel)GetService(typeof(SComponentModel));
             _interactiveWindowProvider = _componentModel.DefaultExportProvider.GetExportedValue<TVsInteractiveWindowProvider>();
+            KnownUIContexts.ShellInitializedContext.WhenActivated(() =>
+                _componentModel.GetService<HACK_ThemeColorFixer>());
```
Comment (ManishJayaswal) [inline src/VisualStudio/InteractiveServices/Interactive/VsInteractiveWindowPackage.cs]: @genlu does this change only affect the pop ups?

Comment (genlu) [inline src/VisualStudio/InteractiveServices/Interactive/VsInteractiveWindowPackage.cs]: @ManishJayaswal based on the code of `HACK_ThemeColorFixer `, it affects both tooltip and immediate , although I don't know what an immediate is...

**Comment by genlu** [comment]
@dotnet-bot retest this, please

**Comment by amcasey** [comment]
:+1:, assuming you manually verified that it works.

File: src/VisualStudio/InteractiveServices/Interactive/VsInteractiveWindowPackage.cs (hunk 1 above)
Comment (dpoeschl) [inline src/VisualStudio/InteractiveServices/Interactive/VsInteractiveWindowPackage.cs]: The "immediate" ```IClassificationFormatMap``` that we're updating is for the Immediate Window (eventually mapping down to [```DefGuidList.guidImmediateWindowFontCategory```](https://msdn.microsoft.com/en-us/library/vstudio/microsoft.visualstudio.editor.defguidlist.guidimmediatewindowfontcategory(v=vs.110).aspx).

**Comment by dpoeschl** [comment]
:+1:

**Pull request merged**